dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java

```diff
@@ -0,0 +1,75 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import java.util.List;
+
+/**
+ * Remove {@code load_records_to_index} stored procedure
+ *
+ * @author nollymar
+ */
+public class Task05225RemoveLoadRecordsToIndex extends AbstractJDBCStartupTask {
+
+    private final String POSTGRES_SCRIPT = "DROP FUNCTION IF EXISTS load_records_to_index;";
+
+    private final String MYSQL_SCRIPT = "DROP PROCEDURE IF EXISTS load_records_to_index;";
+
+    private final String MSSQL_SCRIPT =
+            "IF EXISTS (SELECT * FROM sys.objects WHERE type = 'P' AND name = 'load_records_to_index')\n"
+                    + "DROP PROCEDURE load_records_to_index;";
+
+
+    private final String ORACLE_SCRIPT = "DROP FUNCTION load_records_to_index";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5198932)